source/brailleTables.py

```diff
@@ -788,6 +885,10 @@ def initialize():
 					"Error while applying custom braille tables config from scratchpad manifest: "
 					f"{manifestPath}",
 				)
+	if config.conf["braille"]["translationTable"] == "auto":
+		config.conf["braille"]["translationTable"] = getDefaultTableForCurLang(TableType.TRANSLATION)
+	if config.conf["braille"]["inputTable"] == "auto":
```

Comment:
I would handle this in the brailleInput handler instead. Recall my comment about being a bit confused that there were no changes to brailleInput.py? Here's the clarification.